roles-and-groups.md

```diff
@@ -0,0 +1,153 @@
+- **Evaluate your project's performance**: Committers and Maintainers should monitor their project health and velocity and make decisions about whether the project needs more Committers.
+- **Verify your project's needs**: Committers and Maintainers should define the number of good contributions that a Contributor must reach to consider a nomination. This is based on the size and complexity of the project in question.
+- **Nominate new committers**: To promote a contributor, a vote should be held in a GitHub PR (by proposing a change to the config.yaml file) and by describing the basis reached for nominating the Contributor. The template [vote_pr_template.md](https://github.com/hiero-ledger/governance/blob/main/.github/PULL_REQUEST_TEMPLATE/vote_pr_template.md) should be used to creating a nomination.
+- **Approve the nomination**: All Maintainers should vote on that nomination and the vote passes when a majority of maintainers vote to proceed. The voting procedure is explained in the same template [vote_pr_template.md](https://github.com/hiero-ledger/governance/blob/main/.github/PULL_REQUEST_TEMPLATE/vote_pr_template.md).
+- **Make adjustments**: Maintainers are responsible for removing inactive committers. A committer is considered inactive if there has been no activity for six months and there has been no response when trying to contact them.
+As long as a Committer follows our Code of Conduct, makes reasonable contributions to the project,
+and is not violating any of our policies, they will keep their Committer role.
+
+Project maintainers may remove a committer if:
+
+- Breaks Code of Conduct.
+- Shown no activity for 6 months and does not respond when contacted by the project maintainers or LFDT staff.
+- Keeps making bad decisions (their approvals and contributions are negatively affecting the project's performance).
+
+The addition or removal of committers must be decided by a project maintainer vote. The voting must be held via a GitHub PR (by proposing a change to the `config.yaml` file).
+
+### Maintainers
+
+The Maintainers of the project are the people that are responsible for the project.
+
+A Maintainer should drive the project's roadmap/vision and take care of topics like community meetings,
+the general structure of the project (fits our best practices, is secure, ...), and so on.
+
+In addition, the Maintainers should report project updates to the Hiero **Technical Steering Committee** (TSC) meeting.
+A Committer of a project can be nominated to become a Maintainer:
+
+- **Define your project's needs**: A repository should have 1 to 3 maintainers to ensure a project's sustainability and resilience. It is not necesarily a goal for the committer to become a maintainer unless the project's needs require it to have more maintainers.
```

Comment:
Again the phrase / introduction before the list does not fell well to the points of the list.

Reply:
This relates to my comment I made in my previous note, this doc is organized in roles sections. In this case, we have one addressed to the Committers and one addressed to the Maintainers. Their guidelines are similar and might look repetitive but is important to keep them isolated to offer more flexibility.

Reply:
We can change it afterwards :) Not a blocker for this PR